README.md

```diff
@@ -116,9 +116,12 @@ _options:_
 * -i    file path to the XLSX for use with xlsx2json
 * -D    (optional) path to the sheets folder; if no folder is specified the the current working directory is used
 
-_[\<sheet name\> ...]:_
+The json2xlsx option take following additional argument:
+* -e    comma delimited list of wildcards patterns of the files which needs to be excluded from processing,
```

Comment:
A`...of the files...`, I am thinking to replace it with `...of file names...` to express it only targets filenames (including expansion), not paths.